src/typo3/customize.md

```diff
@@ -306,86 +308,105 @@ by this Fluid template:
 
 # Multiple shops
 
-Aimeos is multi-site capable and allows storing several shops in one database. In order to tell TYPO3 which shop should be used in what page tree, you have to add a TypoScript configuration for each shop page tree. This is not necessary for the page tree that should contain the first shop (site: "default").
+In *Aimeos* terms, a "shop" is a "site". Therefore, when discussing whether the system can handle "mutlipe shops", we say that *Aimeos* is *multi-site capable*. It allows storing several shops in one database.
 
-## Create new site
+In TYPO3, each shop needs its own TYPO3 page tree, the root of which has to be configured with a special pageTS setting (except the first one).
 
-In the Aimeos configuration settings within the extension manager, you can enter a site code for which a new site will be created. If it already exists, it will get updated with the required entries for the used extension version.
+Follow these steps to create multiple shops/sites:
 
-![Aimeos extension settings](Aimeos-settings.png)
+1. **Create a new TYPO3 page**
```

Comment:
This will create an order list which isn't what you want in that case I think

Reply:
Quite the opposite: a ordered list as a step-by-step guide after the text "Follow these steps to ...".